src/plugins/management/public/management_sections.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiToolTip, EuiIcon } from '@elastic/eui';
+
+import { ManagementSectionId } from './types';
+
+interface Props {
+  text: string;
+  tip: string;
+}
+
+const ManagementSectionTitle = ({ text, tip }: Props) => (
+  <EuiToolTip content={tip} position="right">
+    <EuiFlexGroup alignItems="center" gutterSize="s">
```

Comment:
```suggestion
    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
```

Reply:
Thanks for the great spot! Fixed.

![image](https://user-images.githubusercontent.com/1238659/81751334-15b44e80-9464-11ea-9bdf-eb5a9b1c8928.png)